tests/pageObjects/collection-page.ts

```diff
@@ -0,0 +1,91 @@
+import { type Page, type Locator, expect } from '@playwright/test';
```

Comment:
@dualcnhq can we please rename the folder name to `page-objects` instead of `pageObjects`?

Reply:
updated